tool/tsh/common/tsh.go

```diff
@@ -1243,9 +1243,7 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 
 	workloadIdentityCmd := newSVIDCommands(app)
 
-	vnetCmd := newVnetCommand(app)
-	vnetAdminSetupCmd := newVnetAdminSetupCommand(app)
-	vnetDaemonCmd := newVnetDaemonCommand(app)
+	vnetCommands := newVnetCommands(app)
```

Comment:
Do we need to do the tool/tsh changes in this PR? Could this be a prereq or a follow up?